src/snowflake/cli/plugins/snowpark/common.py

```diff
@@ -18,28 +20,25 @@ def check_if_replace_is_required(
     current_state,
     handler: str,
     return_type: str,
+    packages: List[str],
     imports: List[str],
     stage_artifact_file: str,
 ) -> bool:
     import logging
 
     log = logging.getLogger(__name__)
     resource_json = _convert_resource_details_to_dict(current_state)
-    anaconda_packages = resource_json["packages"]
+    deployed_packages = resource_json["packages"]
     log.info(
         "Found %s defined Anaconda packages in deployed %s...",
```

Comment:
```suggestion
        "Found %d defined Anaconda packages in deployed %s...",
```
Nit which we can fix using this occasion

Reply:
done

Reply:
done